api/client/samples/crop_w_series/soybean_w_drought_index.ipynb

```diff
@@ -0,0 +1,745 @@
+{
```

Comment:
<p>You can mention in 1. that <code>rank_series_by_source</code>  was utilized here, and insert a hyperlink to its documentation: https://github.com/gro-intelligence/api-client/blob/development/groclient/lib.py#L422</p><p>Similarly for <code>get_data_points</code></p><p> https://github.com/gro-intelligence/api-client/blob/development/groclient/lib.py#L523<span class="ql-cursor">﻿</span></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gro-intelligence/api-client/pull/311/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>I used these links instead: </p><p> </p><p>https://developers.gro-intelligence.com/api.html#groclient.GroClient.rank_series_by_source</p><p> </p><p>https://developers.gro-intelligence.com/api.html#groclient.GroClient.get_data_points</p><p> </p><p>Let me know if you agree</p>